apps/wallet/src/ui/app/components/accounts/LockUnlockButton.tsx

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { LockLocked16, LockUnlocked16 } from '@mysten/icons';
+import { type ComponentPropsWithoutRef } from 'react';
+import { Tooltip } from '../../shared/tooltip';
+
+interface LockUnlockButtonProps extends ComponentPropsWithoutRef<'button'> {
+	isLocked: boolean;
+}
+
+export function LockUnlockButton({ onClick, isLocked }: LockUnlockButtonProps) {
```

Comment:
nit: should we just spread all props besides `isLocked` into `button` since our props interface has `extends ComponentPropsWithoutRef<'button'>`?